starter-code/index.html

```diff
@@ -6,9 +6,46 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
   <link rel="stylesheet" href="./css/style.css">
-  <script type="text/javascript" src="./js/index.js"></script>
   <title>Ironhack cart</title>
 </head>
 <body>
+<h1>Ironhack Merchandising Shop</h1>
+
+<div>
+<div class= "product">
+  <div class = "product-name">IronBubble-head</div>
+  <div class= "one-unit-cost">$25.50</div>
```

Comment:
Here, and for the quantity, you've been told to create a `<div>` with a `<span>` inside.
Adding the span would have allowed you to isolate the numeric value of the price and the quantity, saving you the pain of trying to do slices and splices later in your function. Like so:

```html
  <div> $ <span class= "one-unit-cost">25.50</span> </div>
```